node_modules/@stdlib/iter/cartesian-power/README.md

```diff
@@ -0,0 +1,212 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# iterCartesianPower
+
+> Create an iterator which generates the Cartesian power of an input array-like object.
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var iterCartesianPower = require( '@stdlib/iter/cartesian-power' );
+```
+
+#### iterCartesianPower( x, n )
+
+Returns an iterator which generates the Cartesian power of an input array-like object.
+
+```javascript
+var x = ['a', 'b', 'c'];
+var n = 2;
+var iterator = iterCartesianPower( x, n );
+
+for (const combination of iterator) {
```

Comment:
Everything should be written in ES5. See also https://github.com/stdlib-js/stdlib/blob/develop/FAQ.md#es2015-and-beyond.

Reply:
You can perform manual iteration. See other iterator packages for examples.